daffodil-test/src/test/scala/org/apache/daffodil/section07/assertions/TestAssertions.scala

```diff
@@ -98,6 +98,4 @@ class TestAssertions {
   @Test def test_assertWithMessageExpression_01() { runner.runOneTest("test_assertWithMessageExpression_01") }
   @Test def test_assertWithMessageExpression_02() { runner.runOneTest("test_assertWithMessageExpression_02") }
 
-  //DFDL-1210
-  //@Test def test_assertFailShowsValue() { runner.runOneTest("assertFailShowsValue") }
```

Comment:
Was this removed by accident? Looks like the test is defined in assert.tdml.

Reply:
I think it was.  I actually  needed to reopen DAFFODIL-752 for this issue, as the extra detail in assertion failed messages seems to have been lost.